packages/harpguru-core/src/components/harp-faces/harp-faces.tsx

```diff
@@ -0,0 +1,44 @@
+import { useGlobal } from 'reactn'
+import { View, StyleSheet } from 'react-native'
+import React from 'react'
+import { isChromaticHarpFace } from 'harpparts'
+
+import { HarpFace } from '../harp-face'
+import { useSizes } from '../../hooks'
+
+export const HarpFaces = (): React.ReactElement => {
+  const [fullInteractionMatrix] = useGlobal('activeInteractionMatrix')
+  const {
+    dynamicSizes: { fragmentGutter },
+  } = useSizes()
+  const { stack, gutter } = StyleSheet.create({
+    stack: {
+      ...StyleSheet.absoluteFillObject,
+      flexDirection: 'column',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    gutter: {
+      height: fragmentGutter,
+    },
+  })
+  // TODO: Have this as part of the layout facts
```

Comment:
The problem with this is that at the moment we are only spending a rerender of this component when the interaction matrix is updated, but the LayoutFacts might be updated more regularly than that and produce unnecessary renders.

There is also the problem that having a property in global state will not give us the same typeguarding assurances that actually using the typeguard here will. There is more potential for misalignment.

Reply:
This has been mentioned in the following issues:

- https://github.com/js-jslog/harpguru/issues/134
- https://github.com/js-jslog/harpguru/issues/135